magento/magento2
# Add indexer console command output

add additional id column to bin/magento indexer:status command in order to have that information ready

### Description (*)
in order to ease the process of the unregular event an indexer has to be reindexed by a console command on a (e.g. live-)server i want to see which indizes are not in status ready and reindex them with the second command bin/magento indexer:reindex <<indexer-id>> which is now available with the status command

### Manual testing scenarios (*)

1. connect to server / run terminal locally
2. go to magento root folder
3. call bin/magento indexer:status
4. check if according id is shown as first column (it can be checked against the bin/magento indexer:info command)

### Contribution checklist (*)
